### DISCUSSION TITLE
svelte-scoped regression in Svelte 4

### DISCUSSION THREAD

**Discussion by scheatkode** [unocss/unocss#2811] [Q&A]
I think I hit an edge case in Svelte 4, where certain components are excluded from the generated styles (searching for the component hash yields no results). I made sure that there are no additional hashes present, ruling out the possibility of hash mismatches.

A workaround to address this issue is to downgrade Svelte to 3.59.2.

I will open a proper issue once I'm able to provide a reproduction. @jacob-8, any pointers ?

**Comment by jacob-8** [comment]
@scheatkode No pointers yet. Post a repro when you can :)

**Comment by scheatkode** [comment]
Opened #2818.
